src/modules/search/searchSagas.js

```diff
@@ -36,24 +36,44 @@ export function* watchSearch() {
   }
 }
 
-export function* searchDraft(query) {
+export function* searchAudio(query) {
   try {
     const searchResult = yield call(api.searchDraft, query);
-    yield put(actions.setDraftSearchResult({ results: searchResult }));
+    yield put(actions.setAudioSearchResult({ results: searchResult }));
   } catch (error) {
     yield put(actions.searchError());
     // TODO: handle error
     console.error(error); // eslint-disable-line no-console
   }
 }
 
-export function* watchSearchDraft() {
+export function* searchImage(query) {
+  try {
+    const searchResult = yield call(api.searchDraft, query);
```

Comment:
Det gamle mediasøket går via search-api for å gruppere søket, men dette trengs ikkje lenger. Det er gjerne like greit å gå direkte på image-api og audio-api på samme måte som vi allerede gjør for forklaringer.